### PR TITLE
Feature/muwm 4643

### PR DESCRIPTION
**Pull request by fanglinfang**
https://jira.cac.washington.edu/browse/MUWM-4643
Hold the merge.
We can't release this before pronouns are released in the Identity.uw.edu UI, and the project team/sponsors approve and set a go live date.

### CODE REVIEW

File: myuw/views/api/finance.py [hunk 1]
```diff
@@ -28,9 +28,8 @@ def get(self, request, *args, **kwargs):
 
             balances = get_account_balances_for_current_user(request)
 
-            date = get_tuition_due_date(request)
             response = balances.json_data()
-            response['tuition_due'] = str(date)
+            # response['tuition_due'] = str(get_tuition_due_date(request))
```
Comment (fanglinfang) [inline myuw/views/api/finance.py:32]: Skip this as we get the tuition due date from notice in JS.

**Review by devights — APPROVED** [review]

**Review by lmanes — CHANGES_REQUESTED** [review]
https://github.com/uw-it-aca/myuw/pull/2254/files/3021b21e6f2930d6b4fc651dce879acb7731d339#diff-2a52f1d1adf4602766c2340013cf99b3f5aa8607547e0d2214c97a65e4f183b5 

myuw/templates/handlebars/teaching/photo_list.html line 153 (on the new version)

We had to add this so the string would wrap in the photo grid version of the list. (Also had to increase the margin on the `last-child` `li` (https://github.com/uw-it-aca/myuw/pull/2254/files?file-filters%5B%5D=.html&file-filters%5B%5D=.less#diff-76b69079bb201d437f406e9a45b4e7118514e4fac1148f569711012a01f0e97dR71) 

However, if the student's name is two lines long and the pronoun string is two lines long, it's still truncating and cutting off text. 

image

I mocked this up by overriding as a different instructor -- someone with classes with lots of students, not just javerage. Then I went into dev tools and added a pronouns string into `div class="myuw-text-small"` for a whole bunch of students -- I used the same string for all of them: `She/her/hers; they/them/their/theirs/themself`

I tried adding padding and/or margin to several different elements (the `li`, the `div`) to see if it would fix this, but it didn't. I'm guessing there might be something about the overflowing text not actually increasing the height of the element, making it not behave like we expect? Just a guess. At this point it needs more investigation. Let me know if I can help guide you.

File: myuw/templates/handlebars/teaching/photo_list.html [hunk 2]
```diff
@@ -148,6 +150,7 @@ <h4 class="sr-only">Grid of Student Photos</h4>
                                  src="/photo/{{url_key}}" width="120px" height="150px">
                              </div>
                             <div class="myuw-photo-list-content">{{first_name}} {{surname}}</div>
+                            <div class="myuw-text-small" style="overflow-wrap: break-word;">{{pronouns}}</div>
```
[the remark this answers is not in the document]
Comment (kbui98) [reply myuw/templates/handlebars/teaching/photo_list.html:153]: Thanks for making those additions Lauren! I made edits to the pronouns wrapping on the photo grid page. This issue should be resolved now. I'll do another pull request!

**Review by lmanes — APPROVED** [review]
Looks good, thanks!

**Pull request closed without merging**